config/artsey.dtsi

```diff
@@ -225,15 +232,17 @@
 		 *****************************************/
 		// Function layer doesn't apply to non-big artsey presently
 		#if defined ARTSEY_BIG
-		combo_layer_functions { key-positions = <KEY_F_TOP KEY_F_BOTTOM>; bindings = <&sl LAYER_ID_BIG_FUNCTION>; };
+		combo_layer_functions { layers = <GLOBAL_COMBO_LAYER_FILTER>; key-positions = <KEY_F_TOP KEY_F_BOTTOM>; bindings = <&sl LAYER_ID_BIG_FUNCTION>; };
```

Comment:
For the combo definitons and generally speaking, GLOBAL_COMBO_LAYER_FILTER, we should probably take a more flexible approach and allow each combos filters to be overridden...

Would you be willing to work on a refactor of that nature? Making all the combo filters overridable and having a few "standard" ones like we have currently and assigning the defaults accordingly.